docs/quickstarts/nextjs.mdx

```diff
@@ -113,20 +113,13 @@ export const config = {
 
 With this middleware, your entire application is protected. If you try to access it, the middleware will redirect you to your Sign Up page. If you want to make other routes public, check out the [authMiddleware reference page](/docs/references/nextjs/auth-middleware).
 
+### Build your own sign-in and sign-up pages
 
+At this point, your application is fully protected by Clerk and uses Clerk's [Account Portal](/docs/account-portal/overview) pages that are available out of the box. You can start your Next.js application via `npm run dev`, visit `http://localhost:3000`, and sign up to get access to your application. 

```

Comment:
💭 I feel like this paragraph makes more sense in the previous section.

Reply:
We used to have it as a Callout in the above section, but I moved it down. My thinking was that it ties in with the next paragraph and and why you may or may not need your own sign-in/sign-up routes. The Callout seemed to get missed by many customers. I can certainly move it up.